dockerfile_functions.sh

```diff
@@ -770,9 +770,7 @@ EOI
 }
 
 # Turn on JVM specific optimization flags.
-# Hotspot container support = https://bugs.openjdk.java.net/browse/JDK-8189497
-# OpenJ9 container support = https://www.eclipse.org/openj9/docs/xxusecontainersupport/
-# OpenJ9 Idle tuning = https://www.eclipse.org/openj9/docs/xxidletuninggconidle/
+# Hotspot@JDK9: https://bugs.openjdk.java.net/browse/JDK-8189497
 print_java_options() {
 	case ${vm} in
```

Comment:
Note sure if case is even required if hotspot is the only option?

Reply:
I was not sure how to go about this method. Neither the hotspot differentiation nor dealing with java9 is relevant anymore.

Since, aside from openj9, the last time it was necessary only for java9 (about 5ish years ago): how about we drop the whole method `print_java_options`?

Reply:
In an excess of caution I'd keep the Java 9 block (sometimes researchers still build 9)